### PR TITLE
updated codebase to Swift 4

### PR DESCRIPTION
**Pull request by LinusGeffarth**

### CODE REVIEW

File: PMZSwitch/PMZSwitch.swift [hunk 1]
```diff
@@ -118,84 +104,70 @@ internal let animationDuration = CFTimeInterval(0.3)
...
-        if on { // left <- right
-            thumbView.animationProgress = (thumbMaxPosition - desiredFrame.origin.x) / maxThumbOffset
-        } else { // left -> right
-            thumbView.animationProgress = (desiredFrame.origin.x - thumbMinPosition) / maxThumbOffset
-        }
+        
+        //                                 // right -> left
+        thumbView.animationProgress = on ? (thumbMaxPosition - desiredFrame.origin.x) / maxThumbOffset :
+            //                             // left -> right
+            (desiredFrame.origin.x - thumbMinPosition) / maxThumbOffset
```
Comment (kovpas) [inline PMZSwitch/PMZSwitch.swift]: This doesn't look very readable to me. Could you please revert this change?

**Review by kovpas — APPROVED** [review]
Looking good to me! Could you please fix the nit change and I'll merge this in.

**Comment by LinusGeffarth** [comment]
@kovpas reverted

**Comment by kovpas** [comment]
Thank you!
